docker/anet-mssql-linux/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# anet-mssql
+# Maintainers: NCI Agency (VassilIordanov on GitHub)
+# GitRepo: https://github.com/NCI-Agency/anet
+
+FROM ubuntu:16.04
+
+RUN export DEBIAN_FRONTEND=noninteractive && \
+    apt-get update && \
+    apt-get install -y apt-transport-https curl  && \
+    curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - && \
+    curl https://packages.microsoft.com/config/ubuntu/16.04/mssql-server.list | tee /etc/apt/sources.list.d/mssql-server.list && \
+    curl https://packages.microsoft.com/config/ubuntu/16.04/prod.list | tee /etc/apt/sources.list.d/msprod.list && \
+    apt-get update && \
+    ACCEPT_EULA=Y apt-get install -y mssql-server=14.0.600.250-2 mssql-server-fts=14.0.600.250-2 mssql-tools unixodbc-dev locales && \
```

Comment:
Do you really want/need the ACCEPT_EULA there? Because https://github.com/Microsoft/mssql-docker/blob/master/linux/preview/examples/mssql-agent-fts-ha-tools/Dockerfile seems to suggest that just:
`apt-get install -y mssql-server=14.0.600.250-2 mssql-server-fts=14.0.600.250-2 mssql-tools unixodbc-dev locales && \`
should work (the burden of ACCEPT_EULA is on the caller).